### DISCUSSION TITLE
Possibility to organize things under ~/.local/share/not1mm$

### DISCUSSION THREAD

**Discussion by awandahl** [mbridak/not1mm#229] [Ideas]
Thinking loud here. Would it be possible to organize things under ~/.local/share/not1mm$ like having different directories for different setups, for different callsigns, for specific contests or similar. These directories could contain the database file(s), the contest or callsign specific macro files, and even specific wav files for the user or the occasion. Would be convenient.

**Comment by mbridak** [comment]
It already does some of this.
When there is a new user call used either thru config or 'OPON' a directory is created and the base wav files are copied into it.
When operating a new contest, a directory for that contest is created and copies of the base macro files are moved into it.
So you're able to have different sets of macros per contest.

**Reply by awandahl** [reply]
Yeah, I can see that now... That is indeed parts of what I want. To complicate thing, my CQWW and General Logging setups are very different when I work as 5X7W or SE0I, so for me it would make sense to have the contests as subdirectories of the callsign directories, and also to store the db file in the specific directory of the contest. By opening the relevant db file in the relevant directory, everything else will be put in place. But I can't have it all I guess... I like to organize things.
